Tasks/DockerV2/dockerbuild.ts

```diff
@@ -71,5 +68,51 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     return dockerCommandUtils.build(connection, dockerFile, commandArguments, labelArguments, tagArguments, (data) => output += data).then(() => {
         let taskOutputPath = utils.writeTaskOutput("build", output);
         outputUpdate(taskOutputPath);
+
+        const builtImageId = getBuiltImageIdFromDockerBuiltOutput(output);
+        if (builtImageId && builtImageId != ""){
+            shareBuiltImageId(builtImageId);
+        }
     });
 }
+
+function shareBuiltImageId(builtImageId: string) {
```

Comment:
Do you think that this functions should be in utils.ts or here? I put it here because just in case are for functions that can be share with the rest of commands and this functions are focus only on the build command

Reply:
I think it makes sense to not move it to `utils.ts`, as it is build specific.

Although since these functions(`shareBuiltImageId` and `getBuiltImageIdFromDockerBuiltOutput`) will be needed in `DockerV0` and `DockerV1` as well, we should move them to `common-npm-packages`.
I think [containerimageutils](https://github.com/microsoft/azure-pipelines-tasks/blob/master/common-npm-packages/docker-common-v2/containerimageutils.ts) might be the right place for these.

Reply:
https://github.com/microsoft/azure-pipelines-tasks/pull/14804